doc/source/howtos/3nodesec2.rst

```diff
@@ -13,22 +13,25 @@ with Red Hat Enterprise Linux 6.1 64-bit.
 
 To set up |PXC|:
 
-1. Remove any |PXC| 5.5, |PS| 5.5, and |PS| 5.6 packages.
+1. Remove |PXC| and |PS| packages for older versions:
 
-#. nstall |PXC| as described in :ref:`yum`.
+   - |PXC| 5.6, 5.7
+   - |PS| 5.6, 5.7
+
```

Comment:
Are we supporting direct 5.6 -> 8.0 upgrade?

Reply:
No we don't. The context here is:
1. Remove |PXC| and |PS| packages for older versions:

   - |PXC| 5.6, 5.7
   - |PS| 5.6, 5.7
This step assumes that the removal is clean and the term upgrade is not technically applicable to the next steps.